test/server/meta.test.js

```diff
@@ -218,6 +218,60 @@ test('should unpack a valid sdk meta bundle with multiple components', () => {
     }
 });
 
+test('should unpack a valid sdk meta bundle with multiple merchant-id email addresses', () => {
+    const emails = [
+        'test@gmail.com',
+        'foo@bar.com',
+        'test@test.org.uk',
+        'test-test@test.com',
+        'test.test@test.com',
+        'test%2Btest@test.com'
```

Comment:
I wonder if we can assume that `'test+test@test.com` will come over encoded at this point in the flow to `'test%2Btest@test.com'`. Testing with clientsdknodeweb should help inform.